static/app/views/issueList/actions/sortOptions.tsx

```diff
@@ -39,20 +54,20 @@ function IssueListSortOptions({onSelect, sort, query}: Props) {
   ];
 
   return (
-    <CompactSelect
-      size="sm"
-      onChange={opt => onSelect(opt.value)}
-      options={sortKeys.map(key => ({
-        value: key,
-        label: getSortLabel(key),
-        details: getSortTooltip(key),
-      }))}
-      value={sortKey}
-      triggerProps={{
-        size: 'xs',
-        icon: <IconSort size="xs" />,
-      }}
-    />
+    <Feature features={['issue-list-better-priority-sort']}>
```

Comment:
nit: wouldn't need the `Feature` or `getSortOptions` function if you did this at the top of the component:

```ts
const organization = useOrganization();
const hasBetterPrioritySort = organization.features.inculdes('issue-list-better-priority-sort')
const sortKeys = [
  ...(hasBetterPrioritySort ? [IssueSortOptions.BETTER_PRIORITY] : []),
  ...other stuff
]
```